libs/aws/langchain_aws/retrievers/bedrock.py

```diff
@@ -123,11 +150,15 @@ def _filter_by_score_confidence(self, docs: List[Document]) -> List[Document]:
         return filtered_docs
 
     def _get_relevant_documents(
-        self, query: str, *, run_manager: CallbackManagerForRetrieverRun
+        self,
+        query: str,
+        *,
+        run_manager: CallbackManagerForRetrieverRun,
     ) -> List[Document]:
         response = self.client.retrieve(
             retrievalQuery={"text": query.strip()},
             knowledgeBaseId=self.knowledge_base_id,
+            nextToken=None,
```

Comment:
This is redundant and not required. https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/bedrock-agent-runtime/client/retrieve.html

```suggestion
            nextToken=None,
```

Reply:
You are right, I removed it.